utils/avd_test.js

```diff
@@ -1,63 +0,0 @@
-/**
```

Comment:
This script was unused, 2 years old and not working. We use our tests instead to ensure Android is working.